src/modules/dashboard/components/ColonyHome/ColonyMembers/ColonyMembers.tsx

```diff
@@ -151,86 +87,25 @@ const ColonyMembers = ({
     );
   }
 
-  if (!colonyMembers) {
-    return (
-      <div className={styles.main}>
-        <Heading
-          appearance={{ size: 'normal', weight: 'bold' }}
-          text={MSG.title}
-          textValues={{ hasCounter: false }}
-        />
-        <span className={styles.loadingText}>
-          <FormattedMessage {...MSG.reputationFetchFailed} />
-        </span>
-      </div>
-    );
-  }
-
   return (
-    <div className={styles.main}>
-      <NavLink to={membersPageRoute}>
-        <Heading
-          appearance={{ size: 'normal', weight: 'bold' }}
-          text={MSG.title}
-          textValues={{
-            count: colonyMembers.length,
-            hasCounter: true,
-          }}
+    <>
+      <MembersSubsection
+        members={subscribers}
+        bannedMembers={bannedMembers?.bannedUsers || []}
+        colony={colony}
+        isContributors
+      />
+      {(currentDomainId === ROOT_DOMAIN_ID ||
+        currentDomainId === COLONY_TOTAL_BALANCE_DOMAIN_ID) && (
+        <MembersSubsection
+          bannedMembers={bannedMembers?.bannedUsers || []}
+          members={membersWithReputation?.colonyMembersWithReputation}
+          colony={colony}
+          maxAvatars={maxAvatars}
+          isContributors={false}
```

Comment:
Should we show the various subsections if they have 0 members ?

I'd argue that no, but maybe @arrenv in product can chime in

Reply:
I had discussed it with Arren before leaving it in and he confirmed to leave even with 0.